devdoc/sm_requirements.md

```diff
@@ -184,6 +186,8 @@ MOCKABLE_FUNCTION(, int, sm_close_begin, SM_HANDLE, sm);
 
 **SRS_SM_02_047: [** If the state is `SM_OPENED` then `sm_close_begin` shall switch it to `SM_OPENED_DRAINING_TO_CLOSE`. **]**
 
+**S_RS_SM_28_008: [** `callback` passed to `sm_close_begin_internal` shall be allowed to be NULL. **]**
```

Comment:
>sm_close_begin_internal [](http://example.com/codeflow?start=44&length=23)

likely this spec doesn't belong here. Probably not needed at all because `sm_close_begin` will not have a callback (internal call is always using `NULL` and `sm_close_begin_with_cb` must have a non-`NULL` `callback` per 28_002